recipes/udocker/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "udocker" %}
+{% set version = "1.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d3011ba32f1f26802ec1e59b8e17d8952371660e2a07437d4860c8f41240de8f
+  patches:
+    - setup.py.patch
+
+build:
+  number: 0
+  skip: True  # [py<36]
```

This looks like it's pure Python and could be noarch Python?
```suggestion
  noarch: python
```